app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -81,7 +82,8 @@ class StateFragmentPresenter @Inject constructor(
 
   private var showCellularDataDialog = true
   private var useCellularData = false
-  private var explorationId: String? = null
+  private var explorationId: String = ""
```

Comment:
I think that this `explorationId` will not change, so better make it `val` and also use `lateinit`with this.

Reply:
I am initializing it in onCreate(). so defined it var.